qutebrowser/browser/commands.py

```diff
@@ -643,14 +643,23 @@ def scroll_perc(self, perc: {'type': float}=None,
 
     @cmdutils.register(instance='command-dispatcher', hide=True,
                        scope='window', count='count')
-    def scroll_page(self, x: {'type': float}, y: {'type': float}, count=1):
+    def scroll_page(self, x: {'type': float}, y: {'type': float},
+            navigate=None, count=1):
```

Comment:
Do you think this would be useful for other navigation actions other than `next`? If not, I think it'd make more sense as an option - e.g. do something like `next=False` and then the command will have a `-n`/`--next` flag.